examples/evolve_dstream_surf.py

```diff
@@ -123,28 +178,35 @@ def __init__(
             )
 
             # handle phylotrackpy instrumentation...
-            self.uid = make_uuid4_fast()
+            self.uid = make_uuid4_fast(use_selector=not is_fossil)
             self.taxon = syst.add_org(self, parent_taxon)
 
             # handle hstrat surface instrumentation...
             self.hstrat_surface = parent_hstrat_surface.copy()
             # ... deposit stratum...
-            differentia_value = random.randrange(2**differentia_bitwidth)
+            # hack to avoid varying ranges give different random states
```

Comment:
Can we make this a helper function? We could name it `consistent_randrange` or something

Reply:
and still have the comment that it's being done to ensure the RNG stays synchronized

Reply:
There must be a better/more descriptive name we can think of than `consistent_randrange` though haha